scripts/coco

```diff
@@ -214,11 +214,12 @@ if hasattr(parsed_args, "func"):
                     parsed_args.style, STYLES
                 )
             )
+            sys.exit(1)
```

Comment:
Why was this one added in this location with this `else` statement nested within `if result`?

Reply:
This branch is for "Unknown print style", as the print statement says. It was previously continueing to print "Done" and exit(0), but it's an error really.